server/tests/test_db.py

```diff
@@ -283,6 +283,93 @@ async def test_get_ranking(self):
+            mock_uuid.return_value = '2'
+            user2 = await self.conn.authenticate_user('test_user_2', b'password', auto_create=True)
+            mock_uuid.return_value = '3'
+            user3 = await self.conn.authenticate_user('test_user_3', b'password', auto_create=True)
+
+            # Generate 2 matches for user_2 (winner) vs user_3 (loser)
+            for i in range(1, 3):
+                mock_uuid.return_value = f'1{i}'
+                test_match = DbMatchReport(
+                    winner_id=user2.user_id,
+                    loser_id=user3.user_id,
+                    winner_pieces_left=i,
+                    loser_pieces_left=20-i,
+                    move_counter=20,
+                    grid_size='small',
+                    squadron_size='medium',
+                    started_at=datetime(2020, 1, 1, i, 0, 0),
+                    finished_at=datetime(2020, 1, 1, i+1, 0, 0),
+                    is_ranked=True,
+                    is_void=False,
+                )
+
+                await self.conn.add_match_result(test_match)
+
+            # Generate 2 matches for user_2 (winner) vs user_1 (loser)
+            for i in range(1, 3):
+                mock_uuid.return_value = f'2{i}'
+                test_match = DbMatchReport(
+                    winner_id=user2.user_id,
+                    loser_id=user1.user_id,
+                    winner_pieces_left=i,
+                    loser_pieces_left=20-i,
+                    move_counter=20,
+                    grid_size='small',
+                    squadron_size='medium',
+                    started_at=datetime(2020, 1, 1, i, 0, 0),
+                    finished_at=datetime(2020, 1, 1, i+1, 0, 0),
+                    is_ranked=True,
+                    is_void=False,
+                )
+
+                await self.conn.add_match_result(test_match)
+
+            # Generate 5 matches for user_0 (winner) vs user_1 (loser) - user_1 lost some rating on user_2
+            for i in range(1, 6):
+                mock_uuid.return_value = f'0{i}'
+                test_match = DbMatchReport(
+                    winner_id=user0.user_id,
+                    loser_id=user1.user_id,
+                    winner_pieces_left=i,
+                    loser_pieces_left=20-i,
+                    move_counter=20,
+                    grid_size='small',
+                    squadron_size='medium',
+                    started_at=datetime(2020, 1, 1, i, 0, 0),
+                    finished_at=datetime(2020, 1, 1, i+1, 0, 0),
+                    is_ranked=True,
+                    is_void=False,
+                )
+
+                await self.conn.add_match_result(test_match)
+
+            # user_2 played less games than user_1,
+            # but played against different players, therefore should have higher rating
+            ranking_entries = [
+                RankingEntry(username='test_user_2', user_id='2', wins=4, games=4),
```

Comment:
Previous commit also had ratings here, but to have them here would require adding them solely for these tests, they are not used anywhere else

Reply:
TODO:
- [ ] Add this back and ignore it everywhere else